vitest.config.ts

```diff
@@ -9,7 +9,7 @@ export default defineConfig({
 			include: ["src"],
 			reporter: ["html", "lcov"],
 		},
-		exclude: ["lib", "node_modules"],
+		exclude: ["lib", "node_modules", "node_modules", "node_modules"],
```

Comment:
🐛 These should be deduped. Caused by https://github.com/JoshuaKGoldberg/create-typescript-app/pull/2079.